diff --git a/Base/Logic/vtkSlicerBaseLogicExport.h b/Base/Logic/vtkSlicerBaseLogicExport.h
index 75f3bc2137..c500df460f 100644
--- a/Base/Logic/vtkSlicerBaseLogicExport.h
+++ b/Base/Logic/vtkSlicerBaseLogicExport.h
@@ -16,15 +16,6 @@
 #define __vtkSlicerBaseLogicExport_h
 
 #include <vtkSlicerBaseLogicConfigure.h>
-
-#if defined(_WIN32) && !defined(VTKSLICER_STATIC)
-# if defined(SlicerBaseLogic_EXPORTS)
-#  define VTK_SLICER_BASE_LOGIC_EXPORT __declspec(dllexport)
-# else
-#  define VTK_SLICER_BASE_LOGIC_EXPORT __declspec(dllimport)
-# endif
-#else
-# define VTK_SLICER_BASE_LOGIC_EXPORT
-#endif
+#include <vtkSlicerBaseLogicModule.h>
 
 #endif
