diff --git a/Libs/MRML/DisplayableManager/vtkMRMLDisplayableManagerExport.h b/Libs/MRML/DisplayableManager/vtkMRMLDisplayableManagerExport.h
index 708f8c6362..7393c7932f 100644
--- a/Libs/MRML/DisplayableManager/vtkMRMLDisplayableManagerExport.h
+++ b/Libs/MRML/DisplayableManager/vtkMRMLDisplayableManagerExport.h
@@ -15,20 +15,6 @@
 #define __vtkMRMLDisplayableManagerExport_h
 
 #include <vtkMRMLDisplayableManagerConfigure.h>
-
-#if defined(_WIN32) && !defined(MRMLDisplayableManager_STATIC)
-# if defined(MRMLDisplayableManager_EXPORTS)
-#  define VTK_MRML_DISPLAYABLEMANAGER_EXPORT __declspec(dllexport)
-# else
-#  define VTK_MRML_DISPLAYABLEMANAGER_EXPORT __declspec(dllimport)
-# endif
-#else
-# define VTK_MRML_DISPLAYABLEMANAGER_EXPORT
-#endif
-
-#if defined(MRMLDisplayableManager_AUTOINIT)
-# include <vtkAutoInit.h>
-VTK_AUTOINIT(MRMLDisplayableManager)
-#endif
+#include <vtkMRMLDisplayableManagerModule.h>
 
 #endif
