diff --git a/Libs/vtkSegmentationCore/vtkSegmentationCoreConfigure.h.in b/Libs/vtkSegmentationCore/vtkSegmentationCoreConfigure.h.in
index 1cb105e9ad..a87d6d035c 100644
--- a/Libs/vtkSegmentationCore/vtkSegmentationCoreConfigure.h.in
+++ b/Libs/vtkSegmentationCore/vtkSegmentationCoreConfigure.h.in
@@ -9,15 +9,4 @@
 #endif
 
 #cmakedefine vtkSegmentationCore_USE_UUID
-
-#if defined(WIN32) && !defined(vtkSegmentationCore_STATIC)
-#pragma warning ( disable : 4275 )
-
-#if defined(vtkSegmentationCore_EXPORTS)
-#define vtkSegmentationCore_EXPORT __declspec( dllexport )
-#else
-#define vtkSegmentationCore_EXPORT __declspec( dllimport )
-#endif
-#else
-#define vtkSegmentationCore_EXPORT
-#endif
+#include <vtkSegmentationCoreModule.h>
